.gitignore

```diff
@@ -1 +1,4 @@
 /tmp/
+/venv/
+/.python-version
+/.ruff_cache/
```

Comment:
ignore ruff caches not only in the repo root but wherever they are 

```suggestion
.ruff_cache/
```

Reply:
done